docs/src/main/asciidoc/chapters/administration.txt

```diff
@@ -1182,17 +1182,19 @@ conditions that vary from the environments individually tested in the various
 components. For example, Accumulo's use of HDFS includes many short block
 reads, which differs from the more common full file read used in most
 map/reduce applications. We have found that certain versions of Accumulo and
-Hadoop will include stability bugs that greatly affect overall stability. In
-our testing, Accumulo 1.6.2, Hadoop 2.6.0, and Zookeeper 3.4.6 resulted in a
-stable VM clusters that did not fail a month of testing, while Accumulo 1.6.1,
-Hadoop 2.5.1, and Zookeeper 3.4.5 had a mean time between failure of less than
-a week under heavy ingest and query load. We expect that results will vary with
-other configurations, and you should choose your software versions with that in
-mind.
-
-
-
-
-
-
-
+Hadoop will include stability bugs that greatly affect overall stability.
+Release notes typically contain information about versions used in
+release testing.
+
+The following table shows the Hadoop, Zookeeper and
+Thrift versions defined in the dependency section of the POM for building the
+artifacts.  
+
+|================================================
+|Accumulo |Hadoop |Zookeeper |Thrift
+|1.7            |2.2.0     |3.4.6          |0.9.1
+|1.8            |2.6.4     |3.4.6          |0.9.3
+|================================================
+
```

Comment:
I'm not sure we need this section in the user manual. It's also possible we'll change the version in a bugfix release, and this table isn't sufficiently granular (nor do I wish to update it on every release). Wouldn't it suffice to point the user to the dependency management section of the POM, rather than copy the information here?